### PR TITLE
Cb 13

### PR DESCRIPTION
**Pull request by mrabiabrn**
I added  the error handler file.

### CODE REVIEW

File: practice-app/app.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 from flask import Flask, jsonify
 import datetime
 from database import mongo
+from .errorHandlers import errorHandlers_bp
```
Comment (CodeBlo) [inline practice-app/app.py:4]:
```suggestion
from errorHandlers import errorHandlers_bp
```
This should also work. Relative imports creates some problems when unittesting.

**Review by asi345 — DISMISSED** [review]
Seems to be working really well.

**Review by hamzaakyildiz — DISMISSED** [review]
Works properly

**Review by kocasariumut — DISMISSED** [review]
works as intended

File: practice-app/app.py [hunk 2]
```diff
@@ -210,42 +273,13 @@
...
 def index():
     return jsonify({'Uygulama calisiyor mu': 'evet', 'En iyi grup': 'Grup 7'})
 
-#@app.errorhandler(404)
-#def not_found(error):
-#    return make_response(jsonify({'error': 'Task was not found'}), 404)
+
 
 if __name__ == "__main__":
+    fil
```
Comment (ramazanbulut) [inline practice-app/app.py:284]: what does it mean

Comment (CodeBlo) [reply practice-app/app.py:284]: I think it was a remaining of the file_handler. We can delete it

Comment (CodeBlo) [reply practice-app/app.py:284]:
```suggestion
    
```

**Review by hamzaakyildiz — APPROVED** [review]
great !!

**Review by CodeBlo — APPROVED** [review]

**Review by onurcanavci — APPROVED** [review]

**Review by hamzaakyildiz — APPROVED** [review]

**Pull request merged**